kolibri/utils/cli.py

```diff
@@ -593,6 +594,10 @@ def main(args=None):  # noqa: max-complexity=13
         port = _get_port(arguments['--port'])
         check_other_kolibri_running(port)
 
+        # Check if the content directory exists when Kolibri runs after the first time.
+        if os.path.exists(version_file()):
+            check_content_directory_exists()
```

Comment:
Instead, can it be added in `_first_run()` ?

https://github.com/lyw07/kolibri/blob/1b37ca8ab4a7b16b009b257a151933ecd53f2174/kolibri/utils/cli.py#L182